### DISCUSSION TITLE
Is it possible to show all trail marks?

### DISCUSSION THREAD

**Discussion by qlibp** [LeonHeidelbach/trailblazer.nvim#48] [Q&A]
Currently, only 2 marks are shown. Can we show all marks?

**Comment by LeonHeidelbach** [comment]
>  Currently, only 2 marks are shown. Can we show all marks?

What do you mean? All trail marks should be shown in the QuickFix list. Please provide further details and maybe some screenshots. Also as mentioned in your other discussion, please add your config.

**Reply by qlibp** [reply]
I mean the UI just show two marks, I wanna to make it show all marks within current stack.
The quicklist does contain all the marks.

**Reply by LeonHeidelbach** [reply]
I'm not sure what you mean by that. The quickfix list already shows all trail marks of the current stack even if they are spread over multiple buffers. All trail marks for the current buffer should be visible within the text itself by the corresponding stack background highlight color and a symbol in neovims symbol column. Where else would you want to see a list of all trail marks?

**Reply by qlibp** [reply]
Once I run setup() function, it works as expected. Thx for the reply.
